tools/ci_build/github/pai/rocm-ci-pipeline-env.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rocm/cupy:rocm5.5.0_ubuntu20.04_py3.8_pytorch2.0.0_cupy13.0.0
+FROM rocm/cupy:rocm5.6.0_ubuntu20.04_py3.8_pytorch2.0.0_cupy13.0.0
```

Comment:
@PeixuanZuo I recall that you deliberately downgraded the image from 5.6 to 5.5 previous. What was the reason at that time?

Reply:
Need this fix(https://github.com/microsoft/onnxruntime/pull/16735) before update to ROCm5.6

Reply:
Same as this Dockerfile.